can't replace local headers with <>

```
$ ./single_iwyu.py -c /android0/kernel-all/compile_commands.json -s /android0/kernel-all/fs/ext4/fast_commit.c
...
fs/ext4/fast_commit.c:176:10: error: 'fast_commit.h' file not found with <angled> include; use "quotes" instead
  176 | #include <fast_commit.h>
      |          ^~~~~~~~~~~~~~~
      |          "fast_commit.h"
```

--- a/lib/remove_quotes.py
+++ b/lib/remove_quotes.py
@@ -3,15 +3,22 @@
 import sys
 def replace_quotes_with_angle_brackets(text):
     '''Replaces odd " with < and even " with >'''
-    result = []
     replace_with = '<'
-    for char in text:
-        if char == '"':
-            result.append(replace_with)
-            replace_with = '>' if replace_with == '<' else '<'
+    text = text.split('\n')
+    formatted = []
+    for line in text:
+        if '/' in line and '..' not in line:
+            result = []
+            for char in line:
+                if char == '"':
+                    result.append(replace_with)
+                    replace_with = '>' if replace_with == '<' else '<'
+                else:
+                    result.append(char)
+            formatted.append(''.join(result))
         else:
-            result.append(char)
-    return ''.join(result)
+            formatted.append(line)
+    return '\n'.join(formatted)
 
 if __name__ == "__main__":
     print(replace_quotes_with_angle_brackets(sys.stdin.read()))

